README.md

```diff
@@ -48,8 +48,32 @@ module.exports.issueRefreshTokenForUser = function(token) {
 module.exports.validateRefreshToken = function(accessToken, refreshToken) {
   return Promise.resolve(tokens);
 };
+
+// set the token payload issued by login
+module.exports.payloadBuilder = function (user, payload) {
+  return tokenPayloadObject;
```

Comment:
Expand example. This variable doesn't exist. Something like payload.foo = 'bar'; and then return payload.